.gitignore

```diff
@@ -23,3 +23,8 @@
 target/*
 .idea/compiler.xml
 .idea/misc.xml
+.idea/sonarIssues.xml
```

Comment:
@PaulMicu 
Hi Paul, as you can see I tried to exclude them in the gitignore but it has failed me every single time...